### ISSUE TITLE
Have you find out any solutions for the fp16 error? 

### ISSUE THREAD

**Issue by Wunaiq** [zmy1116/timesfromer_trtfp16_error#1]
Have you find out any solutions for the fp16 error? 

BTW, did you speed up the model with fp32 mode memorably?

**Comment by zmy1116** [comment]
no, i suppose they fixed the issue in new tensorrt version but i have not had a chance to verify it

**Comment by Wunaiq** [comment]
Thx for the reply! 
Did you distinctly accelerate the model with fp32/fp16 mode at that time? and what's your GPU version？
